docs/dev/core/retrofit-wrapper.md

```diff
@@ -0,0 +1,39 @@
+# What is the Retrofit Wrapper?
+
+The Retrofit Wrapper is Java library that allows communication with a running instance of the TOSCAna transformer by calling the methods provided by the library.
```

Comment:
Maybe stick with the term `server` (instead of `transformer` - it got never coined AFAIK)